docs/CONTRIBUTING.md

```diff
@@ -133,9 +135,76 @@ Please label issues with the following labels:
 2. `D*`  issue difficulty, suggesting the level of complexity this issue has. AT MOST ONE ALLOWED.
 3. `T*`  Issue topic. MULTIPLE ALLOWED.
 
-## Releases
-
-Declaring formal releases remains the prerogative of the project maintainer(s).
+## Release Process
+
+We are aiming for a **two week** release process of the `polkadot-sdk` repository. The output of a release
+are the Parity `polkadot` node implementation, the runtimes for the Westend & Rococo and new versions 
+of the crates. Given the cadence of two weeks, there is no need to halt a release process for any kind 
+of PR (no BRAKES for the release train). There is only to be made an exception for high security issues 
+for code that is already running in production. However, these critical bug fixes may warrant an out of 
+band release any way, but this should be decided based on the severity on a case by case basis.
+
+### Versioning
+
+We are releasing multiple different things from this repository in one release, but 
+we don't want to use the same version for everything. Thus, in the following we explain
+the versioning story for the node, Westend & Rococo and the crates. The version associated
+to a particular release is taken from the node version.
+
+#### Node
+
+The versioning of the node is done 99% of the time by only incrementing the `minor` version. 
+The `major` version is only bumped for special releases and the `patch` can be used for an 
+out of band release that fixes some critical bug. The node version is not following SemVer. 
+This means that the version doesn't express if there are any breaking changes in the CLI 
+interface or similar. The node version is declared in the `NODE_VERSION` variable in 
+`polkadot/node/primitives/src/lib.rs`.
+
+#### Westend & Rococo
+
+For the test networks we only increment the `spec_version`. The spec version is also following
+the node version. So, `10020` is for example the node release `1.2.0`. This versioning has no
+further meaning and is only done to map from an on chain `spec_version` easily to the 
+release in this repository.
```

Comment:
We have already not followed this. Rococo and Westend Relays are on `103000` (with `1.3.0` release) and the system paras are on `1_003_000` (using the Fellowship "style"). Can we just use the Fellowship style for consistency, `M_mmm_ppp`?

Reply:
I don't care. We can do this.